package.json

```diff
@@ -222,6 +229,7 @@
     "eslint-plugin-promise": "4.2.1",
     "eslint-plugin-standard": "4.0.1",
     "fs-extra": "9.0.1",
+    "globby": "^11.0.0",
```

Comment:
`glob` is already a dependency. Can we use that?

Reply:
Good point, done.